operator/cmd/run/run.go

```diff
@@ -169,6 +170,7 @@ func Command() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	cmd.Flags().StringVar(&secureMetricsAddr, "secure-metrics-bind-address", ":8443", "The address the metric endpoint binds to over a secure port.")
```

Comment:
I'm not sure why this change is included

Reply:
Thanks for catching that, it was unintended due to switching around a bunch of branches, I'll drop it as I update the SHA.